oak_protobuf_idl_tests/test_schema.proto

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.protobuf_idl.tests;
+
+message LookupDataRequest {
+  bytes key = 1;
+}
+message LookupDataResponse {
+  bytes value = 1;
+}
+
+message LogRequest {
+  string entry = 1;
+}
+message LogResponse {}
+
+service TestService {
+  // method_id: 15
```

Comment:
I noticed that the generator uses this information in generating code. This is not required by the gRPC spec though. What happens if the comments are not present?

Reply:
Correct, we use this instead of the method name to simplify things. Currently the generator emits an error if there is not exactly one such comment per rpc method, so hopefully users will figure out what they need to do to fix it (but I'm sure the UX can be improved).